lian/src/lian_nav2/src/mission.cpp

```diff
@@ -3,22 +3,23 @@
 #include "std_msgs/msg/string.hpp"
 #include "nav_msgs/msg/Path.hpp"
 #include "geometry_msgs/PoseStamped.hpp"
+#include "nav_msgs/msg/OccupancyGrid.hpp"
 #include "tf2_ros.h"
 #include <list>
 
-Mission::Mission(const char* fName) : fileName(fName), search(nullptr), logger(nullptr) {}
-
+Mission::Mission(OccupancyGrid occupancyGrid_msg, search(nullptr), logger(nullptr) {}
```

Comment:
`occupanyGrid_msg` - we need a better name, either go for camelCase, or snake_case, but not both

Reply:
For ros msgs the convention for msg names is PascalCase, and without the `msg` part

Reply:
interesting...I wonder what's the motivation for it